Consider armv7ve define

when using -march=armv7ve, clang emits internal architecture define to be
__ARM_ARCH_7VE__ which we do not check to decide if its armv 7 or not, therefore
add this to check as well.

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
--- a/third_party/tcmalloc/chromium/src/base/arm_instruction_set_select.h
+++ b/third_party/tcmalloc/chromium/src/base/arm_instruction_set_select.h
@@ -42,6 +42,7 @@
 #if defined(ARMV8) || \
     defined(__ARM_ARCH_7__) || \
     defined(__ARM_ARCH_7R__) || \
+    defined(__ARM_ARCH_7VE__) || \
     defined(__ARM_ARCH_7A__)
 # define ARMV7 1
 #endif
